Polish onboarding documentation (#739)
mozilla/jira-bugzilla-integration: README.md

### How to onboard a new project?

1. Submit configuration for your project
   - If you're comfortable opening your own pull request, add an entry for your whiteboard tag (eg. `famous-product`) in the [actions configuration files](config/).
     See [actions documentation](docs/actions.md)
   - If not, submit an issue [here](https://github.com/mozilla/jira-bugzilla-integration/issues/new?assignees=&labels=configuration&projects=&template=new-config.yaml&title=Sync+%3CBugzilla+Product%3E+with+%3CJira+Project%3E), and we'll set up your configuration
1. Grant permissions to the Jira Automation Bot

   - If you are an admin of the Jira project

     - go to your Jira project and open `Project Settings`, then `People`.
     - Select `Add People` and search for `Jira Automation`. If two are listed select the one with the green logo
     - From the `Roles` drop down select `Bots`. Click `Add 1 person`.

>      - Add these permission for the bot

Reviewer: ```suggestion
     - Add these permissions for the bot
```

     ```
        "ADD_COMMENTS",
        "CREATE_ISSUES",
        "DELETE_ISSUES",
        "EDIT_ISSUES"
     ```

>    - If you are not an admin of the Jira project contact the admin to determine how best to request the changes described above

Reviewer: Shall we mention `#jira-support` here?

Author: [4370b48](https://github.com/mozilla/jira-bugzilla-integration/pull/739/commits/4370b48b7e830bda0b3cd015ab467ef72f1ab8be)

> 1. Once your configuration is merged and a release is created, create a bug in Bugzilla and add your whiteboard tag to the bug. Note that the tag must be enclosed in square brackets (eg. `[famous-project]`). You should see an issue appear in Jira

Reviewer: ```suggestion
1. Once your configuration is merged and a JBI release is deployed, create a bug in Bugzilla and add your whiteboard tag to the bug. Note that the tag must be enclosed in square brackets (eg. `[famous-project]`). You should see an issue appear in Jira
```

>    - If you want to start syncing a bug to a Jira issue that already exists, add the issue's link to the `See Also` section of the bug before you add the whiteboard tag

Reviewer: ```suggestion
   - If you want to start syncing a bug to a Jira issue that already exists, add the issue's link to the `See Also` section of the Bugzilla bug before you add the whiteboard tag
```
